### DISCUSSION TITLE
Login session wont allow more than one user. Can't log out

### DISCUSSION THREAD

**Discussion by alecmalloc** [Shelf-nu/shelf.nu#1148] [Q&A]
Hey everyone, yesterday I mentioned I was facing issues with using the docker version of the app.
I was able to get the production build up and running, however I'm still facing some issues.
I have one session on my laptop that I can't log out of, and when I try to log in with other accounts or devices I get the issue:

<img width="650" alt="image" src="https://github.com/Shelf-nu/shelf.nu/assets/109140381/24fadbed-a85b-4624-9751-a7334e67fc1f">

This has also been my biggest issue on docker. Wheras on docker I wasnt even able to log in at all.
Could anyone point me in the right direction ?

Much appreciated :)

Update: Seems to work locally on my proxmox VM. So it must be a port forwarding nginx issue. Will lyk.

**Comment by DonKoko** [comment]
Make sure to check the browser console and the terminal for an error. We shouldn't have any unhandled errors in the app so you should see a clear message. 
Also good to know if you are running production or dev build? If you are running production build, its a good idea to run a `npm run dev` and see if the issue also happens.
